dot/config.go

```diff
@@ -75,13 +75,15 @@ type AccountConfig struct {
 
 // NetworkConfig is to marshal/unmarshal toml network config vars
 type NetworkConfig struct {
-	Port        uint32
-	Bootnodes   []string
-	ProtocolID  string
-	NoBootstrap bool
-	NoMDNS      bool
-	MinPeers    int
-	MaxPeers    int
+	Port           uint32
+	Bootnodes      []string
+	ProtocolID     string
+	NoBootstrap    bool
+	NoMDNS         bool
+	MinPeers       int
+	MaxPeers       int
+	PublishMetrics bool
+	MetricsPort    uint32
```

Comment:
these options aren't necessarily for the network but could be global, so could you move them to `GlobalConfig`?

Reply:
Done